examples/ENS/records.en.js

```diff
@@ -3,39 +3,41 @@ class Token {
 
     constructor(tokenInstance) {
         this.props = tokenInstance;
+        this.props.baseNode = ".eth";
+        this.props.fullName = this.props.ensName + this.props.baseNode;
+        let namehash = require("eth-ens-namehash");
+        web3.action.setProps({
+            nodeHash: namehash.hash(this.props.fullName)
+        });
     }
 
     setup() {
-        let namehash = require("eth-ens-namehash");
-        let ensName = this.props.ensName;
-        let subdomainElement = document.getElementById("subdomain");
-        subdomainElement.addEventListener("keyup", function() {
-            let subdomain = subdomainElement.value.toString().trim().toLowerCase();
-            document.getElementById("subdomainHash").innerHTML = namehash.hash(subdomain + "." + ensName);
-        });
+        if(this.props.emailRecord === "") {
```

Comment:
@James-Sangalli Looks like it should be `undefined` when it's not resolved yet (when `nodeHash` hasn't been provided yet):

```
if (this.props.emailRecord === "" || this.props.emailRecord == undefined) {
```